src/elements/md-content.html

```diff
@@ -37,6 +37,10 @@
         font-size: 16px;
       }
 
+      [slot="markdown-html"] a {
+        text-decoration: underline;
+      }
```

Comment:
https://www.w3.org/TR/WCAG20-TECHS/F73.html - links should be visually distinguishable from regular text. Furthermore, there needs to be enough difference that if someone with low vision / a bad monitor is looking, they can still tell the difference. Adding an underline is a common good way to make it clear what's a link.